lib/core/src/lib/common/utils/date-fns-utils.ts

```diff
@@ -151,4 +153,15 @@ export class DateFnsUtils {
     static parseDate(value: string, dateFormat: string): Date {
         return parse(value, this.convertMomentToDateFnsFormat(dateFormat), new Date());
     }
+
+    /**
+     * Validates a date string using the specified date format.
+     *
+     * @param value - Date string to be matched.
+     * @param dateFormat - The date format string to use for matching.
+     * @returns true if given date string correct against the given format else will return false.
+     */
+    static isMatch(value: string, dateFormat: string): boolean {
+        return isMatch(value, this.convertMomentToDateFnsFormat(dateFormat));
```

Comment:
I think we don't need to use this method here 'this.convertMomentToDateFnsFormat(dateFormat)'.
simply pass the format like this
return isMatch(value, dateFormat);
Update the test case as per the change.

Reply:
The format, to my understanding, should change. Why would you suggest to not change the format?

Reply:
Got it. Thanks.